packages/metrics/src/config/routes.ts

```diff
@@ -691,6 +697,15 @@ export const getRoutes = () => {
         auth: false
       }
     },
+    {
+      method: 'GET',
+      path: '/refreshMetabaseData',
```

Comment:
```suggestion
      path: '/refreshPerformanceData',
```

`Metabase` here is just an implementation detail that the metrics service doesn't need to know anything about